### ISSUE TITLE
Contrast between foreground and background for cookies banner

### ISSUE THREAD

**Issue by agbeltran** [ral-facilities/datagateway#1121]
**Description**:

The cookies banner that appears at the bottom when first loading the page is giving contrast issues. I am including the report below.

----- start report
Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.sgw514)
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

Target application: DataGateway - https://datagateway.isis.stfc.ac.uk/datagateway

Element path: .sgw518 > .sgw514

Snippet: <span class="sgw514">Manage Preferences</span>

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 4.48 (foreground color: #ffffff, background color: #3e863e, font size: 9.8pt (13px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Chrome version 98.0.4758.80

====

This accessibility issue was found using Accessibility Insights for Web 2.30.1 (axe-core 4.3.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

--- end report

**Acceptance criteria**:
- [ ] Fix contrast issue on cookies banner

**Comment by joelvdavies** [comment]
This was closed with https://github.com/ral-facilities/scigateway/pull/959.
